Aquarius/Src/Aquarius/Core/Window.cpp

```diff
@@ -60,6 +60,9 @@ namespace Aquarius {
         m_Context = RenderingContext::Create(m_Window);
         m_Context->Initialize();
 
+        // Initialize Sound
+        m_SoundDevice->get();
```

Comment:
I was wondering how this works but I realized that get is a static method, seems like this class doesn't need to actually have a pointer to a sound device because it doesn't call any methods on it, so this works fine.